packages/rum/src/viewCollection.ts

```diff
@@ -62,10 +70,11 @@ interface ViewContext {
 
 export let viewContext: ViewContext
 
-function newView(
+export function newView(
```

Comment:
This function should only be used internally, you should not export it.
You should find another way to trigger new view in test based on conditions external to `startViewCollection`